src/main/java/com/bugsnag/android/DeviceData.java

```diff
@@ -161,4 +167,40 @@ private static String getAndroidId(Context appContext) {
         ContentResolver cr = appContext.getContentResolver();
         return Settings.Secure.getString(cr, Settings.Secure.ANDROID_ID);
     }
+
+    /**
+     * Gets information about the CPU / API
+     */
+    @NonNull
+    private static String getCpuAbi() {
+        String cpuAbi = Build.CPU_ABI;
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            cpuAbi = SupportedAbiWrapper.getSupportedAbis();
+        } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.FROYO) {
+            cpuAbi = Abi2Wrapper.getAbi1andAbi2();
+        }
+
+        return cpuAbi;
```

Comment:
This method can be simplified to
```java
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
  return SupportedAbiWrapper.getSupportedAbis();
}
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.FROYO) {
  return Abi2Wrapper.getAbi1andAbi2();
}
return Build.CPU_ABI;
```

Also is it worth wrapping `Build.CPU_ABI` in an array so that the output format is always consistent regardless of API level?